plugins/hydra_ray_launcher/setup.py

```diff
@@ -21,6 +21,6 @@
             "Operating System :: MacOS",
             "Operating System :: POSIX :: Linux",
         ],
-        install_requires=["boto3>=1.12.34", "hydra-core==1.0.*", "ray==0.8.5"],
+        install_requires=["boto3>=1.12.34", "hydra-core~=1.0.0rc1", "ray>=0.8.6"],
```

Comment:
Just to be sure, can you confirm that `hydra-core~=1.0.0rc1` is automatically picking up 1.0.0rc2 ?

Reply:
Yes. I've tried that. It works.